lib/kb_Metrics/metricsdb_controller.py

```diff
@@ -58,311 +55,308 @@ def __init__(self, config):
         if not self.mongodb_dbList:  # pragma: no cover
             warnings.warn('no "mongodb-databases" are set in config of MetricsMongoDBController.')
         # give warnings if no mongo user information is set
-        if 'mongodb-user' not in config: # pragma: no cover
+        if 'mongodb-user' not in config:  # pragma: no cover
             warnings.warn('"mongodb-user" is not set in config of MetricsMongoDBController.')
```

Comment:
@qzzhang why not terminating? Is there any legit cases where the program should continue without mongodb-user?